Documentation/targets.md

```diff
@@ -1,30 +1,113 @@
 # Targets
 
+- alertmanager
+- api
+- app-exporter
+- app-operator
 - cadvisor
 - calico
 - cert-exporter
 - chart-operator
 - cluster-autoscaler
+- cluster-operator
+- cluster-service
+- companyd
+- credentiald
+- crsync
 - docker
+- draughtsman
 - etcd
+- etcd-backup-operator
+- fluentbit
+- g8s-grafana
+- g8s-prometheus
+- gatekeeper
 - kube-proxy
 - kube-state-metrics
 - kubelet
 - kubernetes-apiserver
 - kubernetes-controller-manager
 - kubernetes-scheduler
+- kubernetesd
 - net-exporter
 - nginx-ingress-controller
 - node-exporter
+- node-operator
+- organization-operator
+- prometheus-meta-operator
+- release-operator
+- tokend
+- userd
+- vault
+- vault-exporter
 
 ### AWS specific
 
 - aws-node
+- aws-operator
+- cluster-autoscaler
 
-## Known issues
+### Azure specific
 
-* docker: fails to be scrapped, due to metrics port not being exposed on giantswarm releases below 11.0.0, see [commit](https://github.com/giantswarm/k8scloudconfig/commit/6ecc07e665c3e854dfa8be102a8c6446d1d9dc3c#diff-be6122463e3fe598d118a80e09254d3d)
+- azure-operator
+- cluster-autoscaler
+
+### KVM specific
+
+- kvm-operator
+
+## Missing
+
+- bridge-operator
+- cert-operator
+- prometheus-operator-app
+- CP prometheus
+- TC prometheus
+- flannel-operator
+- coredns
+- ingress-exporter
+- azure-collector
+
+## Not scraped
+
+- cert-manager
+- default-http-backend
+- dex
+- dex-k8s-authenticator
+- external-dns
+- g8s-cert-manager
+- g8s-cert-manager-injector
+- oauth2-proxy
+- passage
+- passage-redis
+- tiller
+
+### AWS specific
+
+- admission-controller-unique
+- calico-typha
+- kiam
+- opa-mutator-app
+
+### Azure specific
+
+- azure-admission-controller
+- calico-typha
+
+### KVM specific
+
+- calico-kube-controllers
+- heapster
+- monitoring-influxdb
+
+## Unscrapeable
+
+* happa: not exposing any metrics because
```

Comment:
because what?

Reply:
There should not be any because, its just not exposing metrics :D

Reply:
ok, let's remove this `because` then, I think it makes the sentence weird.

Reply:
Done